html-css/week1/Vadim/Portfolio Project/index.html

```diff
@@ -0,0 +1,423 @@
+                        <p> I worked on customer cases daily, providing comprehensive support through various channels such as
+                            email, phone,
+                            and other methods of communication.
+                            My role involved addressing customer inquiries, resolving issues efficiently, and ensuring a positive
+                            experience
+                            at every touchpoint.
+                            I was committed to delivering excellent support by understanding customer needs, offering tailored
+                            solutions,
+                            and maintaining a high standard of service quality.
+                            Building trust and fostering satisfaction were at the core of my interactions with customers.
+                        </p>
+                    </li>
+                    <li>
+                        <h3>Purchase Dept. / Logistics Management</h3>
+                    <div>
+                        <p>Intrafashion SA - Pink Woman • Full-Time</p>
+                        <p class="date">Feb 2010 - Jun 2011</p>
+                    </div>
+                        <p> In my role in the Purchase Department and Logistics, I managed procurement processes, coordinated with
+                            suppliers
+                            to ensure timely delivery of goods, and monitored inventory levels to meet operational needs.
+                            I was responsible for optimizing logistics operations, streamlining supply chain processes, and ensuring
+                            cost-effective transportation and storage solutions while maintaining quality standards.
+                        </p>
+                    </li>
+                    <li>
+                        <h3>Assisting in Creating a Digital Warehouse</h3>
+                    
+                    <div>
+                        <p>Intrafashion SA - Pink Woman • Part-Time</p>
+                        <p class="date">Aug 2009 - Feb 2010</p>
+                    </div>
+                        <p> I contributed to the development and implementation of a digital warehouse system.
+                            This involved assisting in digitizing inventory management processes, integrating data-driven tools for
+                            tracking
+                            stock levels, and improving overall operational efficiency.
+                            I worked on streamlining workflows, ensuring accurate data entry, and facilitating real-time access to
+                            inventory
+                            information for better decision-making and enhanced supply chain coordination.
+                        </p>
+                    </li>
+                </ul>
+            </article>
+
+            <!-- Skills & Knowledge -->              
+            <article id="skills">
+                <div class="skill">
```

Comment:
Merge those into one, so you don't unnecessarily create extra nesting.
```suggestion
            <section id="skills" class="skills">
```

PS. note that I changed the class to `skills` and changed `article to `section`, too.

Reply:
removed some divs there.